charts/daemonset/values.yaml

```diff
@@ -0,0 +1,93 @@
+global:
+  # -- Cloud specific bindings (options: aws, gcp)
+  cloud: ""
+
+# -- Overrides the full-name given to the deployment resources (default: .Release.Name)
+fullnameOverride: ""
+
+image:
+  repository: "nginx"
+  tag: "latest"
+  # -- Whether the repository is public
+  isRepositoryPublic: true
+  # -- The image pullPolicy to use
+  pullPolicy: "IfNotPresent"
+
+config:
+  command: []
+  #  - "/bin/sh"
+  args: []
+  #  - "-c"
+  #  - "echo 'Environment $(hello_env)! Secret $(username).'"
+
+  # -- Map of environment variables to use within the job
+  env:
+  #  hello_env: "world"
+
+  # -- Map of secrets that will be exposed as environment variables within the job
+  secrets: {}
+  #  username: "password"
+
+# -- List of config maps to mount to the deployment
+configMaps: []
+#  - name: "volume-1"
+#    mountPath: "/etc/config" # Must be unique
+#    mountPropagation: None # If unset will default to 'None'
+#    files: []
+#      - key: "file.cfg" # Key must be unique for each file
+#        contentsB64: "" # The file contents which have already been base-64 encoded
+#        contentsFile: "" # The path to a local file (note: contentsB64 will take precedence if not-empty)
+
+# -- List of host paths to mount to the deployment
+hostPaths: []
+#  - name: "rootfs"
+#    path: "/"
+#    type: "Directory"
+#    mountPath: "/rootfs"
+#    readOnly: true
```

Comment:
since hostPaths are sketch in the first place, could this be forced read only or do we have active read-write scenarios we need the configuration for?

Reply:
We don't currently ... happy to change this for the moment and we can introduce non-RO if we need it.